release_notes.md

```diff
@@ -3,18 +3,18 @@
 ## Enhancement
 
 - Introduced New playbook:
-    - **Ingest Latest CISA ICS Advisory**, playbook ingests all the ICS Advisory record.
-    - **Ingest Latest CISA ICS Medical Advisory**, playbook ingests all the ICS Medical Advisory record.
+    - **Ingest CISA ICS Advisory**, playbook ingests all the ICS Advisory record.
+    - **Ingest CISA ICS Medical Advisory**, playbook ingests all the ICS Medical Advisory record.
     - **Notify for SLA and Report**, playbook notify for BOD 22-01 KEV Alert SLA remaining time and report.
     - **> Check KEV Alert SLA**, playbook check for KEV Alert SLA to notify.
 
 - Introduced New Schedule:
-    - **Ingest Latest ICS Advisory**, triggered daily to ingest ICS Advisory
-    - **Ingest Latest ICS Medical Advisory**, triggered daily to ingest ICS Medical Advisory
+    - **Ingest CISA ICS Advisory**, triggered daily to ingest ICS Advisory
```

Comment:
Ingest PB and schedule name are same. Please check other schedules and confirm if it needs any change of name?

Reply:
As of now we have not shipped more schedule with SP.

Reply:
but schedule name should indicate it as its a schedule?

Reply:
But why to highlight schedule name, since user can see it under Schedule moduels